Dockerfile

```diff
@@ -60,3 +62,19 @@ RUN rm -f /etc/httpd/conf.d/autoindex.conf /etc/httpd/conf.d/welcome.conf /etc/h
 COPY ironic-config/httpd.conf /etc/httpd/conf.d/
 COPY ironic-config/httpd-modules.conf /etc/httpd/conf.modules.d/
 COPY ironic-config/apache2-ironic-api.conf.j2 /etc/httpd-ironic-api.conf.j2
+
+
+# IRONIC-INSPECTOR #
+RUN mkdir -p /var/lib/ironic-inspector && \
+  sqlite3 /var/lib/ironic-inspector/ironic-inspector.db "pragma journal_mode=wal" && \
+  dnf remove -y sqlite
+
+COPY ironic-inspector-config/ironic-inspector.conf.j2 /etc/ironic-inspector/
+COPY ironic-inspector-scripts/ /bin/
+
+COPY ironic-inspector-config/inspector-apache.conf.j2 /etc/httpd/conf.d/ironic-inspector.conf.j2
+HEALTHCHECK CMD /bin/runhealthcheck
+RUN rm /etc/httpd/conf.d/ssl.conf -f
+RUN chmod +x /bin/runironic-inspector
+
+ENTRYPOINT ["/bin/runironic-inspector"]
```

Comment:
Should be removed

Reply:
yep, this needs change in BMO at least as we're still relying on it
https://github.com/metal3-io/baremetal-operator/pull/851